beauty-consulting-app/app/Pages/ProfileView.js

```diff
@@ -1,119 +1,437 @@
-import React, { useContext, useState } from "react";
-import { View, ScrollView } from "react-native";
+// ProfileView.js
+import React, { useContext, useState, useEffect } from "react";
+import {
+	StyleSheet,
+	Text,
+	View,
+	ScrollView,
+	TouchableOpacity,
+	Modal,
+	TextInput,
+	Alert,
+	ActivityIndicator,
+	Image,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
 import { UserContext } from "../contexts/userContext";
 import SignupBackground from "../assets/components/SignupBackground";
-import api from "utils/axios";
```

Comment:
I don't think this should be deleted as this is the new way we are making HTTP requests, as per #89.

Reply:
Added back.